docs/deployment/README.md

```diff
@@ -266,40 +228,72 @@ dev:                            # deployment name
   cmr:
     username: '{{CMR_USERNAME}}'
     password: '{{CMR_PASSWORD}}'
-    clientId: '<replace-with-daac-name>-{{stackName}}'
-    provider: CUMULUS
+    clientId: '<replace-with-daac-name>-{{stackName}}' # Client-ID submitted to CMR to identify origin of requests.
+    provider: CUMULUS                                  # Target provider in CMR
+
+  es:                               # Optional. Set to 'null' to disable elasticsearch.
+    name: myES5Domain               # Optional. Defaults to 'es5vpc'.
+    elasticSearchMapping: 2         # Optional, triggers elasticSearch re-bootstrap.
+                                    # Useful when e.g. mappings are updated.
+
+  app:                              # Override params to be passed to the app stack ('iam' and 'db' also allowed)
```

Comment:
So these are overrides and not merged with the default params?

Reply:
Yes, e.g.: ```  if (this.config.app && this.config.app.params) this.config.params = this.config.app.params;```

Reply:
The deployment package has no `app:` field, just `params`, which are overridden in the updated `lib/kes.js`.